.github/workflows/stale-bot.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+name: 'Close stale issues and PR '
+on:
+  schedule:
+    - cron: '0 20 * * SUN' # every Sunday at 20 am UTC: PST 0:00 AM "
+    #- cron: '* * * * *' # For every minute and only for testing purpose"
+
+
+jobs:
+  stale-close:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Mark the issues/pr
+        uses: actions/stale@v4.0.0
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} #Github workflow will add a temporary token when executing the workflow
+        with:
+          #Github stale actions: https://github.com/actions/stale
+          stale-issue-message: 'This issue is stale because it has been open 25 days with no activity. Remove stale label or comment or this will be closed in 5 days.'
+          stale-pr-message: 'This PR is stale because it has been open 30 days with no activity.'
+          close-issue-message: 'This issue was closed because it has been stalled for 5 days with no activity.'
+          days-before-issue-stale: 25 #Mark the issues as after 25 days
+          days-before-pr-stale: 30 #Mark the PR as stale after 30 days
+          days-before-issue-close: 5 #Close the issue if the issue has been marked as stall for 5 days
+          days-before-pr-close: -1 #Never close down the PR and keep the label stall on the PR
+          debug-only: true #Debugging purpose. Moreover, need to set secret ACTIONS_STEP_DEBUG = true to enable this
```

Comment:
Did you set up the secret in the main repo?

Reply:
Currently, I don't have permission to set up the secrets but will raise this to @alolita.

Reply:
Not yet. I have raised to @alolita and waited for her to respond back